doc/debugger.rst

```diff
@@ -216,6 +216,20 @@ Called when the application wants to attach a debugger to the Duktape heap::
                         my_detached_cb,           /* debugger detached callback */
                         my_udata);                /* debug udata */
 
+Or, if you need to support AppRequest messages::
```

Comment:
Since this is pretty early introductory text, maybe the AppRequest feature could be spelled out here, e.g.:

> Or, if you want to use application specific commands through the AppRequest debug command (see "Custom requests and notifications"):
>  ....